Add deploy script

Zarel opened the pull request.

Zarel committed:
Add deploy script

Zarel commented:
So, this script doesn't work yet. PS crashes with:

```
Error Loading Teambuilder.json Error: ENOENT: no such file or directory, open '/Users/zarel/Workspace/digimon-showdown/pokemon-showdown/config/chat-plugins/teambuilder.json'
    at Object.openSync (fs.js:457:3)
    at Object.readFileSync (fs.js:359:35)
    at FSPath.readSync (/Users/zarel/Workspace/digimon-showdown/pokemon-showdown/.lib-dist/fs.js:66:13)
    at Object.<anonymous> (/Users/zarel/Workspace/digimon-showdown/pokemon-showdown/server/chat-plugins/teambuilder.js:369:66)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
```

This repository seems to be missing `teambuilder.json`.

Dragotic-PS commented:
> This repository seems to be missing `teambuilder.json`.

Should be resolved with #52

Zarel committed:
Automatically deploy with npm run start

Instead of manually needing the `./deploy` script, it will now run
automatically.

Zarel commented:
Yep, it works now.

Now all you need to do to run Digimon Showdown is

```
```
